### DISCUSSION TITLE
API Takes Twice As Long

### DISCUSSION THREAD

**Discussion by stavrosdidakis** [comfyanonymous/ComfyUI#5565] [Q&A]
I am running ComfyUI from the API (from Unity).
My workflows from the native interface take about 70sec to execute, but from the API take more than 600sec.
Does someone know why this may be happening? How can I optimize this to have the same time of execution?

**Comment by ltdrdata** [comment]
You can investigate the issue in this way.
https://github.com/comfyanonymous/ComfyUI/discussions/4831#discussioncomment-10718745
